iot-logistics/time-map/index.html

```diff
@@ -0,0 +1,153 @@
+
+        <!-- Legend if you want it -->
+        <!-- <div class="graphic__legend">
+          <span class="legend-box legend-box-1"></span>
+          <span class="legend-text">Legend text 1</span>
+          <span class="legend-box legend-box-2"></span>
+          <span class="legend-text">Legend text 2</span>
+          <span class="legend-box legend-box-3"></span>
+          <span class="legend-text">Legend text 3</span>
+          <span class="legend-box legend-box-4"></span>
+          <span class="legend-text">Legend text 4</span>
+        </div> -->
+      </div>
+
+      <div id="wallaroo-map">
+        <div class="graphic__legend">
+          <div>
+            <span class="legend-box legend-box-5"></span>
+            <span class="legend-text">Replacement</span>
+          </div>
+          <div>
+            <span class="legend-box legend-box-1"></span>
+            <span class="legend-text">Normal shipment</span>
+          </div>
+          <div>
+            <span class="legend-box legend-box-2"></span>
+            <span class="legend-text">Delayed</span>
+          </div>
+          <div>
+            <span class="legend-box legend-box-3"></span>
+            <span class="legend-text">Damaged</span>
+          </div>
+          <div>
+            <span class="legend-box legend-box-4"></span>
+            <span class="legend-text">Delayed and damaged</span>
+          </div>
+        </div>
+
+        <div class='graphic__input'>
+          <h3>Day: <label id='active-day'></label></h3>
+          <h3>Time: <label id='active-hour'></label></h3>
+          <input id='slider' class='row' type='range' min='3600' max='6480' step='30' value='6480' type="range" />
+
+          <div class="graphic__input--dropdown">
+            <select id="flag-select">
+              <option value="0">All shipments</option>
+              <option value="1">Delayed</option>
+              <option value="2">Damaged</option>
+              <option value="3">Delayed and damaged</option>
+              <option value="4">Replacement</option>
+            </select>
+          </div>
+        </div>
+      </div>
+
+
+
+
+
+
+
+
+
+
+
+      <!-- begin ai2html here DO NOT MODIFY THIS LINE -->
+
+      <!-- end ai2html here DO NOT MODIFY THIS LINE -->
+
+      <div class='source-and-credit bentonsanscond-regular'>
+        <!-- <p class='source'><span>Source: Source</span></p>
+        <p class='credit'><span>Reporter/Globe Staff</span></p> -->
+      </div>
+      <!--
+      ==========================
+      END OF GRAPHIC CODE
+      ==========================
+      -->
+
+    </div>
+
+    <!-- (begin) put js libraries here -->
+
+    <!-- (end) -->
+
+    <!--
+    ============================
+    *** DO NOT GO BELOW HERE ***
+    ============================
+    -->
+
+
+    <!-- (begin) globe iframe embed -->
+    <script src='https://apps.bostonglobe.com/common/js/pym/pym-1.3.2.min.js'></script>
```

Comment:
Is it going to be a problem to pull this from the Boston Globe site rather than hosting it ourselves?

Reply:
This is not needed. I just updated my commit